src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -1,136 +1,136 @@
-        {
-            var order = await _orderRepository.GetOrderByIdAsync(orderId);
-            if (order is null)
-            {
-                return NotFound();
-            }
-
-            var primaryOrganisationId = User.GetPrimaryOrganisationId();
-            if (primaryOrganisationId != order.OrganisationId)
-            {
-                return Forbid();
-            }
-
-            OrderSummaryModel orderSummaryModel = new OrderSummaryModel
-            {
-                OrderId = orderId,
-                OrganisationId = order.OrganisationId,
-                Description = order.Description.Value,
-                Sections = new List<SectionModel>
-                {
-                    SectionModel.Description,
-                    SectionModel.OrderingParty.WithStatus(order.IsOrderingPartySectionComplete() ? "complete" : "incomplete"),
-                    SectionModel.Supplier.WithStatus(order.IsSupplierSectionComplete() ? "complete" : "incomplete"),
-                    SectionModel.CommencementDate.WithStatus(order.IsCommencementDateSectionComplete() ? "complete" : "incomplete"),
-                    SectionModel.AssociatedServices,
-                    SectionModel.ServiceRecipients.WithStatus(order.IsServiceRecipientsSectionComplete() ? "complete" : "incomplete"),
-                    SectionModel.CatalogueSolutions,
-                    SectionModel.AdditionalServices,
-                    SectionModel.FundingSource
-                }
-            };
-
-            return Ok(orderSummaryModel);
-        }
-
-        [HttpPost]
-        [Authorize(Policy = PolicyName.CanManageOrders)]
-        public async Task<ActionResult<CreateOrderResponseModel>> CreateOrderAsync([FromBody][Required] CreateOrderModel order)
-        {
-            if (order is null)
-            {
-                throw new ArgumentNullException(nameof(order));
-            }
-
-            var primaryOrganisationId = User.GetPrimaryOrganisationId();
-            if (primaryOrganisationId != order.OrganisationId)
-            {
-                return Forbid();
-            }
-
-            var result = await _createOrderService.CreateAsync(new CreateOrderRequest
-            {
-                LastUpdatedByName = User.GetUserName(),
-                LastUpdatedById = User.GetUserId(),
-                Description = order.Description,
-                OrganisationId = order.OrganisationId,
-            });
-
-            var createOrderResponse = new CreateOrderResponseModel();
-
-            if (result.IsSuccess)
-            {
-                createOrderResponse.OrderId = result.Value;
-                return CreatedAtAction(nameof(CreateOrderAsync).TrimAsync(), null, new { orderId = result.Value }, createOrderResponse);
-            }
-
-            createOrderResponse.Errors = result.Errors.Select(x => new ErrorModel(x.Id, x.Field));
-            return BadRequest(createOrderResponse);
-        }
-    }
-}
+﻿using System;
```

Comment:
On Github comparison it looks like the whole file has changed. Please ensure the file contains the expected line endings.

Reply:
The file was CRLF in editor and i thought we were moving to LF, so i made the change?

Reply:
Changed it back to CRLF